### PR TITLE
bump wiremock version to 2.1.11

### PR DESCRIPTION
**Pull request by mlarcher**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,11 +4,11 @@ FROM ekino/java7
 MAINTAINER fronton@ekino.com
 
 ENV WM_PACKAGE wiremock
-ENV WM_VERSION 1.55
+ENV WM_VERSION 2.1.11
```
Comment (plouc) [inline Dockerfile:7]: You should use `ARG` https://docs.docker.com/engine/reference/builder/#/arg to be able to easily change the version

File: Dockerfile [hunk 2]
```diff
@@ -4,11 +4,11 @@ FROM ekino/java7
...
 
 RUN mkdir /$WM_PACKAGE
 WORKDIR /$WM_PACKAGE
-RUN curl -sSL -o $WM_PACKAGE.jar https://repo1.maven.org/maven2/com/github/tomakehurst/wiremock/$WM_VERSION/$WM_PACKAGE-$WM_VERSION-standalone.jar
+RUN curl -sSL -o $WM_PACKAGE.jar https://repo1.maven.org/maven2/com/github/tomakehurst/wiremock/$WM_VERSION/$WM_PACKAGE-$WM_VERSION.jar
```
Comment (plouc) [inline Dockerfile]: The correct repo is `http://repo1.maven.org/maven2/com/github/tomakehurst/wiremock-standalone/2.1.9/`

**Comment by plouc** [comment]
@mlarcher:

```
# --- WIREMOCK ---

FROM ekino/java7:base
MAINTAINER fronton@ekino.com

ENV WM_PACKAGE wiremock
ARG WM_VERSION=1.55

RUN mkdir /$WM_PACKAGE
WORKDIR /$WM_PACKAGE
RUN curl -sSL -o $WM_PACKAGE.jar https://repo1.maven.org/maven2/com/github/tomakehurst/wiremock-standalone/$WM_VERSION/$WM_PACKAGE-standalone-$WM_VERSION.jar

EXPOSE 8080

ENTRYPOINT ["java","-jar","wiremock.jar"]
```

**Comment by mlarcher** [comment]
Bien vu, le retour de vacance ne pardonne pas là :/

**Comment by plouc** [comment]
@mlarcher, thx for your contribution, but I close this PR in favor of https://github.com/ekino/docker-wiremock/pull/3 which allow to define WM_VERSION at build time and add support for travis-ci.

**Pull request closed without merging**